ui/client/actions/notificationActions.js

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import InlinedSvgs from "../assets/icons/InlinedSvgs";
+import Notifications from 'react-notification-system-redux';
+
+export function info(message) {
+  return Notifications.show({
+    message: message,
+    level: 'success',
```

Comment:
level: "information"? I thing we should have another method for success and Warnings :)

Reply:
Done. Both levels looks the same. I think it happens because we don't apply default styles.